docs/web/config.yml

```diff
@@ -51,7 +51,7 @@ layout:
       url: https://twitter.com/IbisData
     - font_awesome: github
       url: https://github.com/ibis-project/ibis/
-  footer_note: "© Copyright 2020, Ibis developers"
+  footer_note: "© Copyright 2021, Ibis developers"
```

Comment:
prob should do 2017-2021 (or whatever start date)

Reply:
Looks like the first commit was 2014, I'll update.

Reply:
Done.